README.md

```diff
@@ -1,2 +1,16 @@
 # on-skupack
 Maintain SKU specific package content such as templates and other static files.
+
+# Quick Start
+* Build SKU package
+
+```
+$ ./build-package.bash <sku_pack_directory> <subname>
+```
+
+* Register a new SKU with the package
+```
+$ curl -X POST --data-binary @sku_pack_directory_subname.tar.gz http://localhost:8080/api/common/skus/pack
```

Comment:
Zach: Could please educate me why should use multipart/form data? Any difference between form and data-binary? I found it worked using data-binary, and it is copied from the sku docs you had written.

Felix

-----Original Message-----
From: zyoung51 [notifications@github.com]
Received: 星期四, 21 4月 2016, 20:36
To: RackHD/on-skupack [on-skupack@noreply.github.com]
CC: Yuan, Felix [Felix.Yuan@emc.com]; Team mention [team_mention@noreply.github.com]
Subject: Re: [RackHD/on-skupack] Update README (#12)


In README.md<https://github.com/RackHD/on-skupack/pull/12#discussion_r60571851>:

> @@ -1,2 +1,16 @@
>  # on-skupack
>  Maintain SKU specific package content such as templates and other static files.
> +
> +# Quick Start
> +* Build SKU package
> +
> +```
> +$ ./build-package.bash <sku_pack_directory> <subname>
> +```
> +
> +* Register a new SKU with the package
> +```
> +$ curl -X POST --data-binary @sku_pack_directory_subname.tar.gz http://localhost:8080/api/common/skus/pack


multipart/form-data should be used and the real location of the tarball should be referenced:
curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly or view it on GitHub<https://github.com/RackHD/on-skupack/pull/12/files/a734055e770b761113e863b4696f0397c3811138#r60571851>